src/main/java/vention/entity/UserFactory.java

```diff
@@ -0,0 +1,24 @@
+package vention.entity;
+
+import com.github.javafaker.Faker;
+import io.github.cdimascio.dotenv.Dotenv;
+
+public class UserFactory {
+  private static final Faker faker = new Faker();
+
+  public static User getExistingUser() {
+    Dotenv dotenv = Dotenv.load();
+    return new User.Builder()
+            .email(dotenv.get("EXISTING_USER_EMAIL"))
+            .password(dotenv.get("EXISTING_USER_PASSWORD")).build();
+  }
+
+  public static User createRandomUser() {
```

Comment:
Let's raname this method to getRandomUser . It's more common to name methods with 'get' inside factory classes